<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="LAPTOP-SQBD1KF4" failures="1" tests="1" name="Default test(failed)" time="46.95" errors="0" timestamp="2023-01-05T19:52:36 IST">
  <testcase classname="TestCases.LoginPageTest" name="verifyLoginLogoTest" time="0.047">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestCases.LoginPageTest.verifyLoginLogoTest(LoginPageTest.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginLogoTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyBotLogoTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyBotLogoTest -->
  <testcase classname="TestCases.LoginPageTest" name="loginToAppTest" time="0.0">
    <ignored/>
  </testcase> <!-- loginToAppTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyTitleTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyTitleTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyLoginPageURLTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyLoginPageURLTest -->
</testsuite> <!-- Default test(failed) -->
